font-codegen/src/fields.rs

```diff
@@ -75,6 +76,23 @@ impl Fields {
         self.fields.iter().map(Field::compile_write_stmt)
     }
 
+    pub(crate) fn iter_compile_default_inits(&self) -> impl Iterator<Item = TokenStream> + '_ {
+        self.fields.iter().filter_map(Field::compile_default_init)
+    }
+
+    /// `Ok(true)` if no fields have custom default values, `Ok(false)` otherwise.
```

Comment:
I don't find it tremendously obvious why you don't just return bool from this description?

Reply:
fair, hope that it's clearer now.